src/testUtils/stubs.ts

```diff
@@ -5,3 +5,4 @@ export const NON_ASCII_AWALA_ENDPOINT = 'はじめよう.みんな';
 export const MEMBER_NAME = 'TEST_MEMBER_NAME';
 export const NON_ASCII_MEMBER_NAME = 'テスト ユーザー';
 export const MEMBER_EMAIL = 'membet@example.com';
+export const MEMBER_ID = '6424ad273f75645b35f9ee79';
```

Comment:
I'd call this something like `MEMBER_API_ID`. "Member id" is a different thing in VeraId.

Reply:
Maybe MEMBER_MONGO_ID?

Reply:
Agreed. That's more accurate.

Reply:
I just approved the PR just in case, but feel free to use that name and I'll re-approve it.